7-layouts/styles.css

```
@@ -0,0 +1,145 @@
+/*Mobile design*/
+.articles {
+    display: flex;
+    flex-flow: column wrap;
+    margin-top: 40px;
+}
+
+.articles .animal {
```

```suggestion
.animal {
```
Keep classes independent of their cascade position on the markup. This way you will avoid your markup to match your styles.